rapl-daemon/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"html"
+	"log"
+	"net/http"
+)
+
+const powercapDir = "/sys/class/powercap/"
+
+// Cap is a payload that is expected from Elektron to cap a node
+type Cap struct {
+	Percentage int
+}
+
+func main() {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "Unsupported endpoint %s", html.EscapeString(r.URL.Path))
```

Comment:
error messages should be lowercase.

Reply:
In this case this is some http content being served back to to the client that made the request. I did make it lower case though.